A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/MFD/A320_Neo_MFD.js

```diff
@@ -447,14 +450,34 @@ class A320_Neo_MFD_MainPage extends NavSystemPage {
             this.map.instrument.bingMap.style.visibility = "hidden";
         }
     }
+    // Show/Hide the route
+    setFlightPlanVisibility(flightPlan) {
+        if (flightPlan != this.showFlightPlan) {
+            this.showFlightPlan = flightPlan;
+            this.map.instrument.attributeChangedCallback("show-flightplan", null, flightPlan ? "true" : "false");
+            if (!flightPlan) {
+                this.map.instrument.showConstraints = false;
+                this.map.removeTopOf();
+            } else {
+                const active = SimVar.GetSimVarValue("L:BTN_CSTR_" + this.screenIndex + "_FILTER_ACTIVE", "number");
+                console.log(active);
+                if (active) {
+                    this.map.instrument.showConstraints = true;
```

Comment:
This is nip picking but why not swapping so that the true actually is true?

Reply:
As it is right now I believe it is more readable and easier to understand. 

If a change occurs in flight plan visibility:
    If there is now no flight plan (off):
        remove CSTR and ToC/ToD from display
    If there is now a flight plan (on):
        only if CSTR is active:
            show constraints on display